### DISCUSSION TITLE
Galaxy candidates, 16 < z < 18

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#291] [General]
https://arxiv.org/abs/2411.13640

**Comment by LifeIsStrange** [comment]
Was about to post it!
>  clear tension with pre-JWST theoretical predictions, extending the over-abundance of galaxies from z∼10 to z∼18.6.

Would be nice to update the prediciton of star formation models and MOND.
Are those only photometric or is the some spectroscopy ? (they seem to mention balmer breaks), as there already were Z=20 photometric candidates.
Moreover I'd like to state that the redshift barrier has already been well broken, the resultsts are 2 years old but gatekeeped/not published yet... as can be seen from the various scheduled high Z observations in JWST cycle 2 and cycle 3

**Reply by RedshiftDrift** [reply]
All are photometric estimates.
Yan et al. http://arxiv.org/abs/2311.15121 have candidates up to $z=21.8$.

Using this cosmo calculator
https://cosmology.info/code/marmet_l/cosmology-calculators.html (a copy of Wright's, but with a second calculator for a static cosmology)
At z = 21.8 the LCDM universe is only 159 Myr-old.  That's not enough time to build a galaxy.
At z = 21.8 the galaxy is 41.3 Gly away in a static universe.

Still waiting for a spectroscopic confirmation with enough resolution to see the Lyman line...

**Comment by LifeIsStrange** [comment]
As you can see figure 6 the z=18.6 galaxy is still permitted by LCDM if and only if there is 100% baryon conversion efficiency to stars and exponential star formation history. 
What is not reported however is that this is only true for H0=70.
Using this cosmo calculator 
https://www.astro.ucla.edu/~wright/CosmoCalc.html
With the value of the hubble tension as backed by JWST and countless probes except cmb/bao data, with H0=73km the age of the galaxy change from
195Myr to 187Myr.
This small difference is it seems sufficient to 100% rule out LCDM empirically, even with the wildest star formation mechanisms since the limiting factor here is dark matter halo formation time.

This should be known as the new Hubble tension where the standard model is ruled out under the most proven value ("infinite" sigmas), which is historic.

Btw > The upper limits reach ∼ 10−6 Mpc−3 mag−1 (for CEERS plus PRIMER) at MUV ∼ −19, in a stark contrast to GLIMPSE where we predict there are ≳ 2000× more galaxies per volume at MUV = −18.

**Comment by LifeIsStrange** [comment]
https://arxiv.org/abs/2412.04834
> This is the highest redshift spiral galaxy discovered with JWST so far

**Reply by LifeIsStrange** [reply]
record already broken
https://arxiv.org/abs/2412.13264

**Reply by LifeIsStrange** [reply]
> an implied baryon-to-star conversion efficiency (ϵ∼0.3) that is 1.5 times higher than even the most efficient galaxies at later epochs

milky way sized at z=5.2!

**Reply by budrap00** [reply]
The "implied" baryon to star conversion ratio is nothing more than a model-fitting exercise. It is only implied by the requirement to grow the galaxy in the standard model time frame. This is now becoming the orthodox way to reconcile fully developed galaxies at distances that otherwise don't fit the model. 

Ethan Siegel used this approach re an even more distant galaxy he characterized as a "red monster". It too was the size of the Milky Way and Siegel attributed its "monstrous" size to an even faster stellar conversion rate. Conveniently, this rapid star formation was theorized to be hidden by an enormous amount of dust. And so the zombie lurches on.

**Reply by LifeIsStrange** [reply]
Agreed but there is observational evidence for high dust density
https://arxiv.org/abs/2410.07340

**Reply by RedshiftDrift** [reply]
@LifeIsStrange from https://arxiv.org/abs/2410.07340
> "implies significant dust attenuation of the broad-line emitting region that is presumably associated with the central AGN." - 

In the example you give the high dust density is limited to the central AGN.  The rest of the galaxy does not have dust, as said in the abstract "their blue UV emission is powered by ***unattenuated*** star formation in the host galaxy."

The amount of dust is evaluated using the ["fundamantal plane"](https://en.wikipedia.org/wiki/Fundamental_plane_(elliptical_galaxies)).  One of the properties used in the fundamantal plane is the galaxy luminosity, which is incorrectly calculated in LCDM cosmology (because redshift is not due to space expansion).  As a result, the star population of a distant galaxy is incorrectly determined and the colour of the galaxy is interpreted as being redder than it actually is.  Dust is invoked to explained the 'apparent' reddening (not to reconcile fully developed galaxies at large distances as hypothesized by @budrap00).

**Comment by LifeIsStrange** [comment]
found this survey quite interesting:
possibly largest survey (500+) of morphological analysis of high Z galaxies 
https://arxiv.org/abs/2412.14970

> **relatively constant merger fraction re-
maining out to z ∼ 12**
Our observed galaxy sizes continue to show a
slightly decreasing trend with redshift, as has
been seen in many previous works with both
HST and JWST.
We further observe an evolution to the size-mass
galaxy relation, with galaxies of a given mass
growing in size as time passes, suggesting galaxies
form with denser structures in the early universe.
We observe a galaxy of given mass log(M∗/M⊙)
= 8 to grow in size by ∼ 53% in ∼ 400 Myr
> We find an excess of round (high b/a) objects in
the early universe, and a deficit of low b/a objects,
with smaller sized objects being rounder

Also corroborate the mysterious finding that early galaxies become increasingly banana shaped (??)
https://arxiv.org/abs/2310.15232
interestingly those prolate shapes might be an empirical falsification of CDM (but not warm dark matter)
https://arxiv.org/abs/2407.16339
https://www.nature.com/articles/s41550-024-02384-8

the universality of our mean/median
composite quasar SED may imply that properties of dust attenuation and/or hydrogen
absorption, if any, in the quasar host galaxies at z ≲ 3 should be similar on average,
independent of redshift and luminosity.

**Reply by RedshiftDrift** [reply]
Again, as [explained above](https://github.com/orgs/a-cosmology-group/discussions/291#discussioncomment-11690647), an incorrect determination of galaxy luminosity in LCDM (too faint by a factor $\sqrt{1+z}$ ) and an incorrect determination of galaxy size in LCDM (too small by a factor $1+z$) make the numbers obtained with the ["fundamantal plane"](https://en.wikipedia.org/wiki/Fundamental_plane_(elliptical_galaxies)) meaningless.  Dust and hydrogen attenuation is incorrectly calculated.

Light-propagation in a transparent medium causes small distortion effects similar to [schlieren](https://en.wikipedia.org/wiki/Schlieren).  The images of distant galaxies are distorted by non-uniformities in the IGM.  Just as the sun is not football-shaped, distant galaxies are not banana-shaped.
However, LCDM ***is*** [going pear-shaped](https://dictionary.cambridge.org/dictionary/english/go-pear-shaped) 🤣

![400wm](https://github.com/user-attachments/assets/b8c7d425-e8f2-4263-b96f-1f6ed6b9edd7)

**Comment by LifeIsStrange** [comment]
Highest redshift pre-main sequence stars discovered
https://arxiv.org/abs/2501.17097
